src/Stratis.Features.FederatedPeg/Events/WalletNeedsConsolidation.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NBitcoin;
+using Stratis.Bitcoin.EventBus;
+
+namespace Stratis.Features.FederatedPeg.Events
+{
+    public class WalletNeedsConsolidation : EventBase
```

Comment:
Probably it will be better to have `Event` in the name of the class. 
Also it's not clear what the `amount` is. Is it required amount to consolidate in 1 utxo? Please add a comment to make it more obvious.

Reply:
>Probably it will be better to have Event in the name of the class.

No, please see naming convention of every other event in the solution.